Library/Homebrew/extend/os/linux/hardware/cpu.rb

```diff
@@ -4,16 +4,22 @@ module Hardware
   class CPU
     class << self
       OPTIMIZATION_FLAGS_LINUX = {
-        native:  "-march=#{Homebrew::EnvConfig.arch}",
         nehalem: "-march=nehalem",
         core2:   "-march=core2",
         core:    "-march=prescott",
         armv6:   "-march=armv6",
         armv8:   "-march=armv8-a",
-      }.freeze
+        ppc64le: "-mcpu=powerpc64le",
+      }
 
       def optimization_flags
-        OPTIMIZATION_FLAGS_LINUX
+        OPTIMIZATION_FLAGS_LINUX.tap do |flags|
+          if ppc?
+            flags[:native] = "-mcpu=#{Homebrew::EnvConfig.arch}"
```

Comment:
Why does PPC use `mcpu` here?

Reply:
`-march` is the x86 options. For ppc, we use `-mcpu`

Ref: https://gcc.gnu.org/onlinedocs/gcc/RS_002f6000-and-PowerPC-Options.html#RS_002f6000-and-PowerPC-Options